portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,30 @@
   </head>
   <body>
     <div id="about-content">
-        <div id = "about">
-            <div id="nav-bar">
-                <a href ="#about">about</a>
-                <a href ="#suggestions">suggestions</a>
-                <a href ="#map">map</a>
+        <div id="nav-bar">
```

Comment:
Nitpick: you should use 2 spaces instead of 4 for indentation. I'm not sure if there's a setting in the editor somewhere to flip this?

https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
updated, found it in the editor